library/common/http/dispatcher.h

```diff
@@ -168,6 +168,8 @@ class Dispatcher : public Logger::Loggable<Logger::Id::http> {
     // TODO: https://github.com/lyft/envoy-mobile/issues/825
     void readDisable(bool /*disable*/) override {}
     uint32_t bufferLimit() override { return 65000; }
+    // Not applicable
+    void setFlushTimeout(std::chrono::milliseconds) override { NOT_IMPLEMENTED_GCOVR_EXCL_LINE; }
```

Comment:
This is actually crashing on launch. You can see the trace in logs.

Reply:
Yep, I actually have it fixed over here: https://github.com/lyft/envoy-mobile/pull/940 - thanks for the callout though.